pennylane/io.py

```diff
@@ -140,12 +293,29 @@ def from_qiskit(quantum_circuit, measurements=None):
             qc.cx(0, 1)
             qc.measure_all()
 
-            measurements = [qml.expval(qml.Z(0)), qml.vn_entropy([1])]
-            quantum_circuit = qml.from_qiskit(qc, measurements=measurements)
+        Then we can create a PennyLane circuit that uses this as a sub-circuit, and performs
+        additional operations conditional on the results. We can also do the standard mid-circuit
+        measurement statistics, like expectation value, on the returned measurements:
+
+        .. code-block:: python
 
             @qml.qnode(qml.device("default.qubit"))
-            def circuit_loaded_qiskit_circuit():
-                return quantum_circuit()
+            def circuit():
+                # apply the QuantumCircuit and retrieve the measurements
+                mid_measure0, m0, m1 = qml.from_qiskit(qc)()
+
+                # conditionally apply an additional operation based on the results
+                qml.cond(mid_measure0==0, qml.RX)(np.pi/2, 0)
+
+                # return the expectation value of one of the mid-circuit measurements, and a terminal measurement
+                return qml.expval(mid_measure0), qml.expval(m1)
+
+        >>> circuit()
+        (tensor(0.5, requires_grad=True), tensor(0.5, requires_grad=True))
+
+        The ``IfElseOp``, ``SwitchCaseOp`` and ``c_if`` classical workflows using mid-circuit
+        measurements can be translated from a Qiskit ``QuantumCircuit``
+
 
         >>> print(qml.draw(circuit_loaded_qiskit_circuit)())
         0: ──H──┤↗├──RZ(0.24)─╭●─┤  <Z>
```

Comment:
Not sure what this bit means.

Reply:
It's a remnant from before I started expanding things. I just left that there at the end as a reminder to insert a few drawn circuits into the text, they seem like nice illustrations 😅

Reply:
I've expanded this section and included an example. Let me know what you think!